stories/Tokens/utils/TypographyGenerator.tsx

```diff
@@ -0,0 +1,219 @@
+import jsonBase from '@norges-domstoler/dds-design-tokens/dds/tokens/Base.json';
+import jsonC from '@norges-domstoler/dds-design-tokens/dds/tokens/Core.json';
+import jsonP from '@norges-domstoler/dds-design-tokens/dds/tokens/Public.json';
+
+import { copyButton } from './CopyButton';
+import {
+  generateTypographyBaseTable,
+  isReferencedValue,
+  splitReferenceKeys,
+} from './functions';
+import {
+  type TokenTypographyBaseJsonObject,
+  type TokenTypographySemanticCssFont,
+  type TokenTypographySemanticJsonObject,
+} from './Tokens.types';
+import {
+  type DdsTheme,
+  Table,
+} from '../../../packages/dds-components/src/index';
+
+const baseTokens: Record<string, TokenTypographyBaseJsonObject> = {};
+baseTokens['dds-font-size'] = jsonBase['dds-font-size'];
+baseTokens['dds-font-family'] = jsonBase['dds-font-family'];
+baseTokens['dds-font-weight'] = jsonBase['dds-font-weight'];
+baseTokens['dds-font-lineheight'] = jsonBase['dds-font-lineheight'];
+baseTokens['dds-font-paragraph-spacing'] =
+  jsonBase['dds-font-paragraph-spacing'];
+baseTokens['dds-font-letter-spacing'] = jsonBase['dds-font-letter-spacing'];
+baseTokens['dds-font-style'] = jsonBase['dds-font-style'];
+
+export const TypographyGenerator = (theme: DdsTheme) => {
+  const tokenSet = theme === 'core' ? jsonC : jsonP;
+  const tokens: TokenTypographySemanticJsonObject = tokenSet['dds-font'];
+  const fontSizeReferenceTokens: TokenTypographyBaseJsonObject =
+    tokenSet['dds-font-size'];
+
+  function generateBodyRows() {
+    const rows: Array<React.JSX.Element> = [];
+
+    for (const key1 in tokens) {
+      for (const key2 in tokens[key1]) {
+        const token = tokens[key1][key2];
+        const cssFontValuesObj: TokenTypographySemanticCssFont = {
+          fontWeight: token.value.fontWeight,
+          fontSize: token.value.fontSize,
+          lineHeight: token.value.lineHeight,
+          fontFamily: token.value.fontFamily,
+        };
+        const cssFontValuesArray = Object.values(cssFontValuesObj);
+
+        const values = cssFontValuesArray.map(v => {
+          if (!isReferencedValue(v)) {
+            return v;
+          }
+          const referenceKeys = splitReferenceKeys(v);
+
+          //To lag med referanse; bør endre JSON-strukturen for lettere henting
```

Comment:
Er det opprettet en oppgave for dette v/?

Reply:
Gjorde det nå: https://domstol.atlassian.net/jira/software/c/projects/ELSA1/boards/357?selectedIssue=ELSA1-528